biz.aQute.junit/src/aQute/junit/ConsoleReporter.java

```diff
@@ -0,0 +1,53 @@
+package aQute.junit;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import aQute.lib.exceptions.Exceptions;
+import junit.framework.AssertionFailedError;
+import junit.framework.Test;
+import junit.framework.TestListener;
+
+public class ConsoleReporter implements TestListener {
```

Comment:
Why isn't this just done as part of BasicTestReport class? I don't see much value in making it a separate TestListener that is always added.